c10/core/TensorImpl.h

```diff
@@ -651,6 +651,12 @@ struct C10_API TensorImpl : public c10::intrusive_ptr_target {
     return is_contiguous_default(memory_format);
   }
 
+  inline IntArrayRef strides_default() const {
+    return c10::IntArrayRef(
+        reinterpret_cast<const int64_t*>(sizes_and_strides_.strides_data()),
+        sizes_and_strides_.size());
+  }
```

Comment:
Made this public so it could be called from `concrete_strides_fn` if the return is `None`